qiskit_experiments/calibration/calibrations.py

```diff
@@ -0,0 +1,438 @@
+
+                return qubits[0]
+
+            if isinstance(chan, ControlChannel):
+                indices = [int(sub_channel) for sub_channel in chan.index.name.split(".")]
+                ch_qubits = tuple(qubits[index] for index in indices)
+                chs_ = self._config.control(ch_qubits)
+
+                if len(chs_) < control_index:
+                    raise CalibrationError(
+                        f"Control channel index {control_index} not found for qubits {qubits}."
+                    )
+
+                return chs_[control_index].index
+
+            raise CalibrationError(f"{chan} must be a sub-type of {PulseChannel}.")
+
+        return chan.index
+
+    def get_parameter_value(
+        self,
+        param: Union[Parameter, str],
+        qubits: Tuple[int, ...],
+        schedule: Union[Schedule, str] = None,
+        valid_only: bool = True,
+        group: str = "default",
+        cutoff_date: datetime = None,
+    ) -> Union[int, float, complex]:
+        """
+        Retrieve the value of a calibrated parameter from those stored.
+
+        Args:
+            param: The parameter or the name of the parameter for which to get the parameter value.
+            qubits: The qubits for which to get the value of the parameter.
+            schedule: The schedule or the name of the schedule for which to get the parameter value.
+            valid_only: Use only parameters marked as valid.
+            group: The calibration group from which to draw the
+                parameters. If not specifies this defaults to the 'default' group.
+            cutoff_date: Retrieve the most recent parameter up until the cutoff date.
+                Parameters generated after the cutoff date will be ignored. If the
+                cutoff_date is None then all parameters are considered.
+
+        Returns:
+            value: The value of the parameter.
+
+        Raises:
+            CalibrationError: if there is no parameter value for the given parameter name
+                and pulse channel.
+        """
+        param_name = param.name if isinstance(param, Parameter) else param
+        sched_name = schedule.name if isinstance(schedule, Schedule) else schedule
+
+        if (sched_name, param_name) not in self._parameter_map:
+            raise CalibrationError(f"No parameter for {param_name} and schedule {sched_name}.")
+
+        param = self._parameter_map[(sched_name, param_name)]
+
+        if qubits not in self._params[param]:
+            raise CalibrationError(f"No parameter value for {param} and qubits {qubits}.")
+
+        if valid_only:
+            candidates = [p for p in self._params[param][qubits] if p.valid]
+        else:
+            candidates = self._params[param][qubits]
+
+        candidates = [candidate for candidate in candidates if candidate.group == group]
+
+        if cutoff_date:
+            candidates = [val for val in candidates if val.date_time <= cutoff_date]
```

Comment:
We want values later than the cutoff, right?

```suggestion
            candidates = [val for val in candidates if val.date_time >= cutoff_date]
```